_posts/2019-08-08-hacking-for-cruise-control.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title:  "Hacking Strimzi for Cruise Control"
+date: 2019-08-08
+author: kyle_liberti
+---
+
+Cruise Control is a highly customizable Kafka partition balancing and monitoring software run in production at Linkedin. Cruise Control’s component pluggability makes it flexible to customize for different environments. This makes it possible to extend Cruise Control to work with Strimzi in an Kubernetes environment with a reasonable amount of effort. This post will walk through how to get Cruise Control working with a Stimzi deployed Kafka cluster.
```

Comment:
_Cruise Control’s component pluggability makes it flexible to customize for different environments._ This sounds to me more like part of conclusion. At this point I do not know anything about any Cruise Control components. So maybe just _Cruise Control is flexible and easy to customize for different environments._?

(This is where sentence per line make life a looot easier ;-))